fhirstarter/fhirstarter.py

```diff
@@ -415,6 +415,19 @@ def openapi(self) -> dict[str, Any]:
 
             # Iterate over all operations for a given path
             for operation_name, operation in path.items():
+                # Skip operations that weren't created by FHIRStarter
+                operation_id = operation.get("operationId", "")
+                if not operation_id.startswith("fhirstarter|"):
+                    continue
```

Comment:
I moved this check to the top of the loop, because really none of this processing should be happening for non-FHIRStarter operations.